### PR TITLE
Took over the RDFa initial context in JSON-LD

### PR DESCRIPTION
**Pull request by iherman**
As agreed on the [call on 1st of May](https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2020/2020-05-01-json-ld#section3) here is the RDFa initial context copied in the repository.

Note that, before this step, I have also added the (missing) `jsonld` prefix to the [RDFa initial context](https://www.w3.org/2011/rdfa-context/rdfa-1.1.html), hence to this context file as well.

### CODE REVIEW

**Comment by gkellogg** [comment]
I added the rdfa-1.1.html file to this PR, the build will be done if PR #11 is merged to master first.

If we want to add `@type` information for terms, we could consider (ab)using `rdfs:range` on those terms to trigger setting up `@type`.

**Comment by iherman** [comment]
> I added the rdfa-1.1.html file to this PR, the build will be done if PR #11 is merged to master first.

oops, I did not see this before I commented on #10 (in https://github.com/w3c/json-ld-rc/pull/11#issuecomment-623980493). Then on the contrary, that one should be merged first!

File: context.jsonld [hunk 1]
```diff
@@ -1,66 +1,60 @@
...
-            "@type" : "@id"
-        },
-        "seeAlso": {
-            "@id" : "rdfs:seeAlso",
-            "@type" : "@id"
-        },
-        "HTML": "rdf:HTML",
-        "JSON": "rdf:JSON"
+        "describedby" : "http://www.w3.org/2007/05/powder-s#describedby",
+        "license"     : "http://www.w3.org/1999/xhtml/vocab#license",
```
[the remark this answers is not in the document]
Comment (iherman) [reply context.jsonld:57]: +1 for describedBy, but I would not know how people use role out there. I am almost sure they use it with literal terms...

How would that fit with #10? What would you put into the RDFa file? That would require some `rdf:range`, but would that go through into the context...

[the remark this answers is not in the document]
Comment (gkellogg) [reply context.jsonld:57]: Adding some `rdfs:range` statements could be used by the script that generates the context file to add the `@type` elements to their respective term definitions. But, it’s not enough to destinguish between. `”@type”: “@id”` and `”@type”: “@vocab”`.

**Comment by gkellogg** [comment]
I think we’ll need to push a new commit to this PR to get the workflow to start up.

File: context.jsonld (hunk 1 above)
[the remark this answers is not in the document]
Comment (iherman) [reply context.jsonld:57]: Yep. I would prefer to leave this for a later day...

**Comment by gkellogg** [comment]
Okay, the process failed before, but after running by hand, it passes. We'll need to keep an eye on this.

**Comment by gkellogg** [comment]
@iherman please give a look at the results to see if they look reasonable.

**Comment by iherman** [comment]
> @iherman please give a look at the results to see if they look reasonable.

it absolutely does... I have made a better description in the Readme file; by doing so, I have also tested the PR procedure (see #13)

**Pull request merged**